test/integration/agent_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
You added a `go:build` line, can you tell me why you left the `+build` line?

Reply:
Because of `golangci-lint`. 
If it doesn't support two forms of build constraints, it exited with non-zero value like below.

```
ERRO [linters context] typechecking error: //go:build comment without // +build comment
```

Reply:
Thank you so much for your reply 😄 

Currently, in my local environment(version 1.17), it works without any problem. On the other hand in 1.16 the same issue occurs.

**version 1.17**
![image](https://user-images.githubusercontent.com/40932237/130789218-361f82ea-e7b2-4922-878c-9c3e1114fb71.png)

**version 1.16**
![image](https://user-images.githubusercontent.com/40932237/130789027-f912ff8a-abd7-471a-94f2-d3420f9b99c3.png)

The golangci-lint result is also the same as the editor error.

I thought that updating to 1.17 should improve with the improved code style suggested by 1.17.
Are you considering developers not developing with version 1.17? Please tell us what you think.

Reply:
Thank you! On second thought, it makes sense to do it with an improved code style.